backup_restore_mongo.sh

```diff
@@ -173,19 +201,88 @@ function prep_restore() {
 function restore () {
     title " Restore copy of backup in namespace $TARGET_NAMESPACE "
     msg "-----------------------------------------------------------------------"
-    #change csnamespace to reflect the new target namespace
-    #restore script is setup to look for CS_NAMESPACE and is used elsewhere
+    #export csnamespace to reflect the new target namespace
+    #restore script is setup to look for CS_NAMESPACE and is used in other backup/restore processes unrelated to this script
     export CS_NAMESPACE=$TARGET_NAMESPACE
-    wget https://raw.githubusercontent.com/IBM/ibm-common-service-operator/scripts/velero/restore/mongoDB/mongodbrestore.yaml
-    wget https://raw.githubusercontent.com/IBM/ibm-common-service-operator/scripts/velero/restore/mongoDB/set_access.js
-    wget https://raw.githubusercontent.com/IBM/ibm-common-service-operator/scripts/velero/restore/mongoDB/mongo-restore.sh
+
+    #check if files are already present on machine before trying to download (airgap)
+    if [[ -f "mongodbrestore.yaml" ]]; then
+        info "mongodbrestore.yaml already present"
+    else
+        wget https://raw.githubusercontent.com/IBM/ibm-common-service-operator/scripts/velero/restore/mongoDB/mongodbrestore.yaml
```

Comment:
Do we wanna check the return code of `wget`?

The reason for asking is that if it is a truly airgap env, would it be able to download?

Reply:
The assumption I made for airgap was if it is an airgap environment, the files would need to already exist on the machine before the script runs which the if statement checks for. If the file does not already exist, then it grabs with wget

Reply:
But I checking return code is a good idea. There should be output stating whether or not the items could be downloaded with the response code.